config/_default/menus/menus.en.toml

```diff
@@ -44,7 +44,7 @@
 [[cta]]
     identifier = "cta"
     name = "Download"
-    url = "/docs/download/"
+    url = "/docs/install/"
```

Comment:
Per conversation, let's create a top level /download page for download instructions. It's going to a document detail page with only markdown text for now. 

replace all /docs/install with just /download